postgres-appliance/scripts/configure_spilo.py

```diff
@@ -557,6 +557,8 @@ def get_dcs_config(config, placeholders):
                                 'port': placeholders['EXHIBITOR_PORT']}}
     elif 'ETCD_HOST' in placeholders:
         config = {'etcd': {'host': placeholders['ETCD_HOST']}}
+    elif 'ETCD_HOSTS' in placeholders:
+        config = {'etcd': {'hosts': placeholders['ETCD_HOST']}}
```

Comment:
Looks like a copy&paste error. There should be `placeholders['ETCD_HOSTS']`

Reply:
new PR is opened: https://github.com/zalando/spilo/pull/356
This one can be closed.